apps/console/src/features/identity-providers/pages/identity-providers.tsx

```diff
@@ -697,6 +612,88 @@ const IdentityProvidersPage: FunctionComponent<IDPPropsInterface> = (props: IDPP
                         data-testid={ `${ testId }-list` }
                     />
                 </ListLayout>
+            ) : (
+                <GridLayout
+                    search={ (
+                        <SearchWithFilterLabels
+                            isLoading= { isIdPListRequestLoading || isAuthenticatorFetchRequestRequestLoading }
+                            searchInput={ (
+                                <AdvancedSearchWithBasicFilters
+                                    fill="white"
+                                    onFilter={ (query: string) => {
+                                        handleConnectionGridFilter(query, null);
+                                    } }
+                                    filterAttributeOptions={ [
+                                        {
+                                            key: 0,
+                                            text: t("common:name"),
+                                            value: "name"
+                                        }
+                                    ] }
+                                    // Only 'eq' and 'sw operations are supported in Authenticators API.
+                                    filterConditionOptions={ [
+                                        {
+                                            key: 0,
+                                            text: t("common:startsWith"),
+                                            value: "sw"
+                                        },
+                                        {
+                                            key: 1,
+                                            text: t("common:equals"),
+                                            value: "eq"
+                                        }
+                                    ] }
+                                    filterAttributePlaceholder={ t(
+                                        "console:develop.features.authenticationProvider" +
+                                        ".advancedSearch.form.inputs.filterAttribute.placeholder"
+                                    ) }
+                                    filterConditionsPlaceholder={ t(
```

Comment:
Can we use this format?
```
condition
    ? if true
    : if false
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3653/commits/13fc653e28ee815a2f5323a1157155f5b1e4b3da